src/CLI/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as yargs from "yargs";
+import { CLI } from "./CLIClass";
+
+const args = yargs
+  .usage(
+    `
+Impuddle
+Universal code manager for fetching whole repos, certain folders or certain files from repositories
+
+Usage:
+    imp --add <url> <remote folders> <local folders>
```

Comment:
to trzeba zrobić nie na flagach tylko na komendach czyli powinno być `imp add <url> <remote folders> <local folders>
`